monkey/monkey_island/cc/models/attack_mitigation.py

```diff
@@ -0,0 +1,23 @@
+from mongoengine import Document, StringField, DoesNotExist
+
+
+class AttackMitigation(Document):
+
+    technique_id = StringField(required=True, primary_key=True)
+    name = StringField(required=True)
+    description = StringField(required=True)
+
+    @staticmethod
+    def get_mitigation_by_technique_id(technique_id: str) -> Document:
+        try:
+            return AttackMitigation.objects.get(technique_id=technique_id)
+        except DoesNotExist:
+            raise Exception("Attack technique with id {} does not exist!".format(technique_id))
+
+    @staticmethod
+    def add_mitigation_from_stix2(mitigation_stix2_data):
```

Comment:
perhaps rename mitigation_stix2_data to something more indicative (CourseOfAction somehow)?

Reply:
Renaming - no, stix2 is a format and CourseOfAction is just an object in that format. Renaming doesn't make sense, as I try to emphasize that this method takes in data in stix2 format.